packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -184,9 +184,9 @@ storiesOf('Picker', module)
     'isQuiet',
     () => (
       <Picker isQuiet label="Test" onSelectionChange={action('selectionChange')}>
-        <Item key="100">One hundred</Item>
-        <Item key="2012">Two thousand and twelve</Item>
-        <Item key="3">Three</Item>
+        <Item key="One hundred">One hundred</Item>
+        <Item key="Two thousand and twelve">Two thousand and twelve</Item>
+        <Item key="Three">Three</Item>
```

Comment:
this set can stay as well, the keys are easy to map to the string

Reply:
Thanks, I'll fix it shortly